core/services/pg/transaction.go

```diff
@@ -7,20 +7,16 @@ import (
 	"time"
 
 	"github.com/getsentry/sentry-go"
+	"github.com/jmoiron/sqlx"
 	"github.com/pkg/errors"
 	"go.uber.org/multierr"
 
-	"github.com/jmoiron/sqlx"
-
 	"github.com/smartcontractkit/chainlink-relay/pkg/logger"
-
 	corelogger "github.com/smartcontractkit/chainlink/v2/core/logger"
 )
 
 type TxOptions struct {
 	sql.TxOptions
-	LockTimeout            time.Duration
-	IdleInTxSessionTimeout time.Duration
```

Comment:
Curious as to why these were added if they are not utilized. Were they never needed or there was a time when we used to use them?

Reply:
IIRC they were used in the past :shrug: